app/src/test/resources/design-document.md

```diff
@@ -0,0 +1,151 @@
+# Design Document
+
+## _Collector's Stash_ Design
+
+## 1. Problem Statement
+Many individuals have a passion for collecting various and unique items, be it comic books, stamps, or other collectibles. They often resort to carrying physical notepads and books to manually document and keep track of their collections. This traditional method is overwhelming and can also be prone to errors, loss, and inefficiency. To address this issue, I am developing a innovative application aimed at providing collectors with a digital solution for managing and preserving their collections.
+
+## Top Questions to Resolve in Review
+1. How many tables do I need?
+2. Do I need to use a GSI (Let's say I use it so they can filter/search for their favorites or such)
+3. What is the main goal vs the stretch goals, but clear on what the end goal looks like realistically
+
+## Use Cases
+U1. _As a user, I would like to store my collectibles_
```

Comment:
Not quite sure how to interpret this one. Would this be the use case that covers adding a book to a collection (without adding it to a series)?

Reply:
Got rid of this did not make sense